src/scripts/main.js

```diff
@@ -358,3 +358,26 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const tableBody = document.querySelector('table > tbody');
+
+if (tableBody) {
+  for (const person of people) {
+    const row = document.createElement('tr');
+
+    [
+      person.name,
+      person.sex === 'm' ? 'Male' : 'Female',
```

Comment:
The gender is being represented as 'Male' or 'Female' based on the `sex` property. Ensure this logic aligns with the task requirements.